### PR TITLE
Kanboard example config

### PR DESCRIPTION
**Pull request by MrOnak**
Added a working example config for [kanboard](https://kanboard.org/)

### CODE REVIEW

File: example_configs/kanboard.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# configuration for Kanboard
```
Comment (nitnelave) [inline example_configs/kanboard.md:1]: Nit: capital letter

File: example_configs/kanboard.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+add these to the kanboard `config.php`
```
Comment (nitnelave) [inline example_configs/kanboard.md:3]: Capital

File: example_configs/kanboard.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+Make sure you adapt `LDAP_SERVER` to the correct LDAP endpoint.
```
Comment (nitnelave) [inline example_configs/kanboard.md:5]: Nit: YOUR_LDAP_SERVER

File: example_configs/kanboard.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+You also might have to change `dc=example,dc=com` to whatever your LLDAP is handling, and maybe change the `lldap_kanboard` group name used in `LDAP_USER_FILTER` to identify users of a valid group of yours.
```
Comment (nitnelave) [inline example_configs/kanboard.md:7]: Can you change `lldap_kanboard` to `kanboard_users` ? The groups starting with `lldap_` are reserved by the implementation for future use.

File: example_configs/kanboard.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+define('LDAP_USERNAME_CASE_SENSITIVE', false);
+define('LDAP_USER_CREATION', true);
+
+define('LDAP_BIND_TYPE', 'user');
+define('LDAP_USERNAME', 'uid=%s,ou=people,dc=example,dc=com');
+define('LDAP_PASSWORD', null);
+
+define('LDAP_USER_BASE_DN', 'ou=people,dc=example,dc=com');
+
+define('LDAP_USER_FILTER', '(&(uid=%s)(memberof=cn=lldap_kanboard,ou=groups,dc=example,dc=com))');
```
Comment (nitnelave) [inline example_configs/kanboard.md:24]: `kanboard_users`

File: example_configs/kanboard.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+define('LDAP_USERNAME_CASE_SENSITIVE', false);
+
+define('LDAP_USER_ATTRIBUTE_USERNAME', 'uid');
+define('LDAP_USER_ATTRIBUTE_FULLNAME', 'cn');
+define('LDAP_USER_ATTRIBUTE_EMAIL', 'mail');
+define('LDAP_USER_ATTRIBUTE_GROUPS', 'memberof');
+define('LDAP_USER_ATTRIBUTE_PHOTO', '');
```
Comment (nitnelave) [inline example_configs/kanboard.md:31]: You can actually use "jpegPhoto" here.

**Review by nitnelave — CHANGES_REQUESTED** [review]
Thanks for the contribution! A couple of details, but overall it looks good.

File: example_configs/kanboard.md (hunk 6 above)
Comment (MrOnak) [reply example_configs/kanboard.md]: implemented the requested changes, thanks for the fast response!

**Review by nitnelave — APPROVED** [review]

**Pull request merged**